packages/editor/src/client/routes/myPeerInfo.tsx

```diff
@@ -0,0 +1,224 @@
+import React, {useState, useEffect} from 'react'
```

Comment:
@penk85 is this file needed? From where can you access this route?

Reply:
@bishoymelek you're right. This should go :+1: